libs/withdraws/src/lib/withdraw-form-container.tsx

```diff
@@ -20,24 +19,22 @@ export const WithdrawFormContainer = ({
 }: WithdrawFormContainerProps) => {
   const variables = useMemo(() => ({ partyId }), [partyId]);
   const { data, loading, error } = useDataProvider({
-    dataProvider: accountsOnlyDataProvider,
+    dataProvider: accountsDataProvider,
     variables,
     noUpdate: true,
   });
-
-  const {
-    data: assets,
-    loading: assetsLoading,
-    error: assetsError,
-  } = useDataProvider({
-    dataProvider: enabledAssetsProvider,
-  });
+  const assets = data
```

Comment:
I think this needs to be more rigourous. A user can only withdraw an asset if their balance in their general account is non-zero so I think we need to add that filter logic in here as well

Reply:
I get this point, I was considering this also, but I think it's too tricky because balance of the account is still changing and for this reason we should provide there additional logic for refreshing balances etc..

Reply:
Sure that makes sense :+1: does the withdrawal window not already do this? Just conscious there may be race condition type bugs where a user could try and withdraw more than they have? Obviously not blocking this PR